src/features/multichain/components/CreateSafeOnNewChain/index.tsx

```diff
@@ -69,27 +69,33 @@ const ReplaySafeDialog = ({
   const customRpc = useAppSelector(selectRpc)
   const dispatch = useAppDispatch()
   const [creationError, setCreationError] = useState<Error>()
+  const [isSubmitDisabled, setIsSubmitDisabled] = useState<boolean>(false)
```

Comment:
This is very similar to the `submitDisabled` variable.
Should we rename this to `isSubmitting` and inverse it?